controllers/api/openstack/vm.py

```diff
@@ -87,60 +102,95 @@ def GET(self, action):
 	flavorList = {}
         imageList = {}
 
+        # Flavor List
 	for flavor in novaClient.flavors.list(detailed = True):
 	    flavorList[flavor.id] = {'name':str(flavor.name), 'vcpus':flavor.vcpus, 'ram':flavor.ram}
-        print flavorList
 
+        # Image List
         for image in novaClient.images.list(detailed = True):
-            imageList[image.id] = {'name':str(image.name)}
-        print imageList
+            try:
+                 imageList[image.id] = {
+                     'name' : str(image.name),
+                     'aq'   : image.metadata[u'aq_managed']
+                 }
+            except:
+                 imageList[image.id] = {
+                     'name': str(image.name),
+                     'aq'  : u'false'
+                 }
 
 	for server in novaClient.servers.list(detailed = True):
-	    print server.name + " - " + server.status
+            cherrypy.log('- ' + server.name + ' - ' + server.status, username)
```

Comment:
is there a mix of tabs and spaces around this line? The indentation looks 1 space to many in a side by side comparison but not when viewing the file as a whole